bigchaindb/backend/rethinkdb/admin.py

```diff
@@ -0,0 +1,118 @@
+from bigchaindb.backend.exceptions import DatabaseOpFailedError
+from bigchaindb.backend.utils import module_dispatch_registrar
+from bigchaindb.backend.rethinkdb.connection import RethinkDBConnection
+
+logger = logging.getLogger(__name__)
+
+register_admin = module_dispatch_registrar(admin)
+
+
+@register_admin(RethinkDBConnection)
+def config(connection, *, table):
+    """Get the configuration of the given table.
+
+    Args:
+        connection (:class:`~bigchaindb.backend.connection.Connection`):
+            A connection to the database.
+        table (str): The name of the table to get the configuration for.
+
+    Returns:
+        dict: The configuration of the given table
+
+    """
+    return connection.run(r.table(table).config())
+
+
+@register_admin(RethinkDBConnection)
+def reconfigure(connection, *, table, shards, replicas,
+                primary_replica_tag=None, dry_run=False,
+                nonvoting_replica_tags=None):
+    """Reconfigures the given table.
+
+    Args:
+        connection (:class:`~bigchaindb.backend.connection.Connection`):
+            A connection to the database.
+        table (str): The name of the table to reconfigure.
+        shards (int): The number of shards, an integer from 1-64.
+        replicas (:obj:`int` | :obj:`dict`):
+            * If replicas is an integer, it specifies the number of
+              replicas per shard. Specifying more replicas than there
+              are servers will return an error.
+            * If replicas is a dictionary, it specifies key-value pairs
+              of server tags and the number of replicas to assign to
+              those servers::
+
+                  {'africa': 2, 'asia': 4, 'europe': 2, ...}
+        primary_replica_tag (str): The primary server specified by its
+            server tag. Required if ``replicas`` is a dictionary. The
+            tag must be in the ``replicas`` dictionary. This must not be
+            specified if ``replicas`` is an integer. Defaults to
+            ``None``.
+        dry_run (bool): If ``True`` the generated configuration will not
+            be applied to the table, only returned. Defaults to
+            ``False``.
+        nonvoting_replica_tags (:obj:`list` of :obj:`str`): Replicas
+            with these server tags will be added to the
+            ``nonvoting_replicas`` list of the resulting configuration.
+            Defaults to ``None``.
+
+    Raises:
+        DatabaseOpFailedError: If the reconfiguration fails due to a
+            RethinkDB :exc:`ReqlOpFailedError` or
+            :exc:`ReqlQueryLogicError`.
+
+    """
+    params = {
+        'shards': shards,
+        'replicas': replicas,
+        'dry_run': dry_run,
+    }
+    if primary_replica_tag:
+        params.update(
+            primary_replica_tag=primary_replica_tag,
+            nonvoting_replica_tags=nonvoting_replica_tags,
+        )
+    try:
+        connection.run(r.table(table).reconfigure(**params))
```

Comment:
yes, that is why there is a task at the top in the description of the PR :smile: